services/notifications/models/notification_order.go

```diff
@@ -1,8 +1,9 @@
 package models
 
 type NotificationOrder struct {
-	ID      string `json:"id"`
-	Success int    `json:"success"`
-	Failure int    `json:"failure"`
-	Time    int64  `json:"time"`
+	ID            string `json:"id"`
+	NumRecipients int    `json:"numRecipients"`
```

Comment:
I'd just call this recipients. Or rename the other fields to numSuccess and numFailure